packages/api-plugin-fulfillment-method-shipping-ups/README.md

```diff
@@ -0,0 +1,70 @@
+# api-plugin-fulfillment-method-shipping-ups
```

Comment:
I am uncomfortable with calling this `shipping-ups` since it doesn't actually implement UPS. I would rather it be called something like "dynamic-rate"

Reply:
As discussed, if we are planning to expand this to a more functional (and less dummy) plugin, then we could try out ups as the first option and retain the name.
If that is not the plan, we can change the name as you have suggested.